src/app/functions/server/breachResolution.ts

```diff
@@ -98,7 +98,9 @@ function appendBreachResolutionChecklist(
   const { verifiedEmails } = userBreachData;
 
   for (const { breaches } of verifiedEmails) {
-    breaches.forEach((b: Breach) => {
+    // Old untyped code, adding type defitions now isn't worth the effort:
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    breaches.forEach((b: any) => {
```

Comment:
`verifiedEmails` is of type `any`. I'm not sure where the data originally comes from (i.e. from the DB or the HIBP API) so I'm not sure what the appropriate type is, but the now-deleted `Breach` type wasn't particularly safer than `any`, since it was just a guess too.

Reply:
Looks like it's from https://github.com/mozilla/blurts-server/blob/319434001d2110ceaf32e4dc259d9c793f12ce9b/src/app/functions/server/getUserBreaches.ts#L35 which has a note that we should deprecate it - @mansaj is that actionable now?

Reply:
`verifiedEmails` should come from HIBP API

Reply:
@rhelmer yes it's actionable. We should work on MNTOR-2021 when we have a chance. 

If I remember correctly, we have another function called `getSubBreaches` or something that basically replaces it

Reply:
Thanks, looks like Kaitlyn was right and that this came from `getAllEmailsAndBreaches` from `breaches.js` - updated in d313659b2 (note that I did have to add one other `as any` - comment justifying that added inline).